udata/core/dataset/models.py

```diff
@@ -477,6 +477,8 @@ class Dataset(WithMetrics, BadgeMixin, db.Owned, db.Document):
 
     featured = db.BooleanField(required=True, default=False)
 
+    contact_points = db.ListField(db.ReferenceField(ContactPoint, reverse_delete_rule=db.PULL))
```

Comment:
We're modeling multiple contact points for one dataset here @agarrone.
It may be useful later if we want to have multiple roles (publisher, legal owner, etc.).

Is this an issue?